src/views/devices.mustache

```diff
@@ -101,6 +167,10 @@
         });
         table.on('draw.dt', function() {
             setTimeout(function () {
+				// On Reload, get the actual preview Size to keep it
+		        previewSize = localStorage.getItem('previewSize') !== null ? localStorage.getItem('previewSize') : '96';
+		        query = `?previewSize=${previewSize}`;
+				table.ajax.url('/api/devices' + query).load();
                 table.ajax.reload(null, false);

```

Comment:
No need to reload again after loading the data above.

Reply:
Its needed because everytime the timeout is kicking in (every 30 secs) it reloads the ajax.
Since the query may be outdated by then, we would need to check it on reload so the actual size is up to date with the stored size

Reply:
It works perfectly fine for me with it commented out.